src/authorizations/reducers/index.test.ts

```diff
@@ -0,0 +1,94 @@
+import {normalize} from 'normalizr'
+import {authsReducer, AuthsState} from 'src/authorizations/reducers'
+import {authSchema} from 'src/schemas'
+import {AuthEntities, Authorization, RemoteDataState} from 'src/types'
+import {
+  addAuthorization,
+  editAuthorization,
+  setAuthorizations,
+  removeAuthorization,
+} from '../actions/creators'
+
+const mockState = (): AuthsState => ({
+  allIDs: ['id01', 'id02'],
+  byID: {
+    id01: {
+      org: 'org01',
+      orgID: 'org01id',
+    },
+    id02: {},
+  },
+  status: RemoteDataState.Done,
+})
+
+describe('the authorizations reducer', () => {
+  it('initialize empty state', () => {
+    const newState = authsReducer(undefined, '' as any)
```

Comment:
is the `as any` casting here neccesary?

Reply:
fixed same way as for alertBuilder reducer